core/src/main/java/com/hcl/ecomm/core/config/DeleteCartItemServiceConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "Magento Cart item delete"
+		+ ""
+		+ " Service Configuration")
+public @interface DeleteCartItemServiceConfig {
+
+	String domainNameDefaultValue = "127.0.0.1/magento2";
```

Comment:
@nitheshkumar-hcl : domain is already available in existing Magento login config

Reply:
Removed this config